js/src/Map.js

```diff
@@ -54,7 +69,8 @@ var LeafletMapModel = widgets.DOMWidgetModel.extend({
         options : [],
         layers : [],
         controls : [],
-        crs: 'EPSG3857'
+        crs: 'EPSG3857',
+        style: 'move'
     }),
 
```

Comment:
I think you need to add a new deserializer for the `style` property around line 101

Reply:
I am not sure what deserializers are and what they are for... is this ok?
``` javascript
serializers: _.extend({
        layers : { deserialize: widgets.unpack_models },
        controls : { deserialize: widgets.unpack_models },
        style: { deserialize: widgets.unpack_models }
    }, widgets.DOMWidgetModel.serializers)
```

Reply:
This should be ok :) serialization is for turning object state (whether it is a Python object or a JavaScript object) into a representation that can be stored or sent to through messaging logic. For instance, here we use JSON representations: we turn Python widget state into JSON representation so that it can be sent from the kernel to the client and vice versa. Deserialization is the opposite operation.
Here you need to tell ipywidgets that style is a widget, so it needs to use a deserializer specific to widgets for this to work :)

Reply:
Here I'm talking about object states because I have widgets in mind. But you can have serializers/deserializer for arrays for instance. You could represent NumPy arrays as a JSON object, and send it to the front-end which will create its own JavaScript object representing the array, given the JSON object.

Reply:
Ok, I think I get the idea.. Is that why we call `.tag(sync=True, **widget_serialization)` in the python side?

Reply:
Yep :) You need that for widget attributes.

Reply:
Does it work now?

Reply:
unfortunately.. nop =(

Reply:
If I do this:
``` javascript
b = Button()
b
```
and then
``` javascript
b.style = {'button_color': 'blue'}
```
it works fine...
but if I do:
``` javascript
m = ipyleaflet.Map()
m
```
and then
``` javascript
m.style = {'cursor': 'crosshair'}
```
it doesn't =(